server/src-rsr/initialise.sql

```diff
@@ -302,7 +302,8 @@ CREATE TABLE hdb_catalog.event_log
   error BOOLEAN NOT NULL DEFAULT FALSE,
   tries INTEGER NOT NULL DEFAULT 0,
   created_at TIMESTAMP DEFAULT NOW(),
-  locked BOOLEAN NOT NULL DEFAULT FALSE,
+  /* when locked IS NULL the event is unlocked and can be processed */
+  locked TIMESTAMP,
```

Comment:
Not sure if this will affect the behaviour, but I think it will be a good idea to use `TIMESTAMPZ` instead of `TIMESTAMP` because `now()` returns a `TIMESTAMPZ` and there may be issues when comparing a `TIMESTAMPZ` value with a `TIMESTAMP` value